csaf_2.0/test/validator/run_tests.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+ORIG_SCHEMA=csaf_2.0/json_schema/csaf_json_schema.json 
+STRICT_SCHEMA=csaf_strict_schema.json
+VALIDATOR=csaf_2.0/test/validator.py
+STRICT_GENERATOR=csaf_2.0/test/generate_strict_schema.py
+TESTPATH=csaf_2.0/test/validator/data/$1/*.json
+EXCLUDE=OASIS_CSAF_TC-CSAF_2_0-2021-6-1-08-01.json
+
+FAIL=0
+
+# go to root of git repository
+cd `dirname $0`/../../..
+
+validate() {
+  printf "%s" "Testing file $1 against schema ${SCHEMA} ... "
+  if python3 $VALIDATOR $SCHEMA $1; then
+    printf "%s\n" SUCCESS
+  else
+    printf "%s\n" FAILED
+    FAIL=1
+  fi
+
+}
+
+test_all() {
+  for i in `ls -1 ${TESTPATH} | grep -v $EXCLUDE`
```

Comment:
@sthagen Any idea for this one? I wasn't able to remove the backticks...

Reply:
`$(ls -1 ${TESTPATH} | grep -v $EXCLUDE)` should do.

Reply:
Done in [c3fb0f5](https://github.com/oasis-tcs/csaf/pull/439/commits/c3fb0f5735858723e901a544a88b7ac35b21c20d).